packages/eth-providers/src/utils/utils.ts

```diff
@@ -330,6 +331,11 @@ export const checkEvmExecutionError = (data: CallReturnInfo): void => {
 
 export const isEvmExtrinsic = (e: Extrinsic): boolean => e.method.section.toLowerCase() === 'evm';
 
+export const isBatchExtrinsic = (extrinsic: GenericExtrinsic<AnyTuple>): boolean => {
+  const { method: { method, section } } = extrinsic;
+  return section.toString() === 'utility' && method.toString().includes('batch');
```

Comment:
this doesn't cover `batchAll` and `forceBatch`
another way is to check if there is `utility.batchCompleted` or `utility.batchCompletedWithErrors` event

Reply:
how about `method.toString().toLowerCase().includes('batch')`?

Reply:
that could work. but it is still better to use events instead of checking the extrinsic name. otherwise for example it won't be able to deal with proxy + batch call